srcpkgs/libfixposix/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'libfixposix'
+pkgname=libfixposix
+version=0.4.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool pkg-config"
+short_desc="Replacements for parts of POSIX consistent across *NIX flavours"
+maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
+license="BSL-1.0"
+homepage="https://github.com/sionescu/libfixposix"
+distfiles="https://github.com/sionescu/libfixposix/archive/v${version}.tar.gz"
+checksum=78fe8bcebf496520ac29b5b65049f5ec1977c6bd956640bdc6d1da6ea04d8504
+
+pre_configure() {
+	autoreconf -fi
+}
+
+pkg_install() {
```

Comment:
pkg_install is only for subpackages. for the main package you're looking for do_install()